### PR TITLE
Weconnect api v2 functioning

### PR DESCRIPTION
**Pull request by Rwothoromo**
Please leave any feed back for any improvements.

@dondrzzy
@soniarmk

### CODE REVIEW

File: app/models/user.py [hunk 1]
```diff
@@ -18,17 +18,18 @@ class User(UserMixin, db.Model):
     username = db.Column(db.String(50), nullable=False, unique=True)
     password_hash = db.Column(db.String(256), nullable=False)
     created_at = db.Column(db.DateTime, default=db.func.current_timestamp())
-    updated_at = db.Column(db.DateTime, default=db.func.current_timestamp())
+    updated_at = db.Column(db.DateTime, default=db.func.current_timestamp(
+    ), onupdate=db.func.current_timestamp())
+    businesses = db.relationship(
+        'Business', order_by='Business.id', cascade='all, delete-orphan')
+    # cascade='all, delete-orphan' will delete
```
Comment (Thegaijin) [inline app/models/user.py]: Remove the commented out code

File: manage.py [hunk 2]
```diff
@@ -21,11 +22,13 @@ def test():
     # return the opposite boolean
     return not result.wasSuccessful()
 
+
 @app.route('/')
-def main():
+def main():  # pragma: no cover
```
Comment (Thegaijin) [inline manage.py]: What's going on here? Is that commented out code or a comment?

File: manage.py [hunk 3]
```diff
@@ -21,11 +22,13 @@ def test():
...
     """Redirect to api endpoints"""
 
     return redirect('/api/v2/')
 
+
```
Comment (Thegaijin) [inline manage.py]: Unnecessary extra space

File: manage.py [hunk 4]
```diff
@@ -21,11 +22,13 @@ def test():
     # return the opposite boolean
     return not result.wasSuccessful()
 
+
```
Comment (Thegaijin) [inline manage.py]: Unnecessary extra space

File: app/models/user.py (hunk 1 above)
Comment (Rwothoromo) [reply app/models/user.py]: Oh, the comments, will surely do.

File: manage.py (hunk 2 above)
Comment (Rwothoromo) [reply manage.py]: That tells coverage to skip the function

File: manage.py (hunk 3 above)
Comment (Rwothoromo) [reply manage.py]: Noted

File: manage.py [hunk 5]
```diff
@@ -12,6 +12,7 @@
 manager = Manager(app)
 manager.add_command('db', MigrateCommand)
 
+
```
Comment (Thegaijin) [inline manage.py]: [Python spacing standards](https://www.python.org/dev/peps/pep-0008/#blank-lines)

File: manage.py (hunk 4 above)
Comment (Rwothoromo) [reply manage.py]: Noted

File: manage.py (hunk 5 above)
Comment (Rwothoromo) [reply manage.py]: Noted

**Pull request merged**